patchdeps.py

```diff
@@ -135,11 +135,11 @@ def print_depends_matrix(patches, depends):
     # Which patches have at least one dependency drawn (and thus
     # need lines from then on)?
     has_deps = set()
-    for p in patches:
+    for j, p in enumerate(patches):
         fill, corner = ("─", "┘") if p in has_deps else (" ", " ")
-        line = str(p)[:80] + "  " + fill * (84 - len(line) + p.number * 2) + corner + " "
+        line = str(p)[:80] + "  " + fill * (84 - len(line) + jr * 2) + corner + " "
```

Comment:
Typo in commit " refactor: Replace Changeset.number by enumerate()" `jr` -> `j` (this code is refactored again later, so this error does not show up in the total change).